include/swift/SIL/SILWitnessVisitor.h

```diff
@@ -24,9 +24,6 @@
 #include "swift/AST/ProtocolAssociations.h"
 #include "swift/AST/Types.h"
 #include "swift/SIL/TypeLowering.h"
-#include "llvm/ADT/SmallVector.h"
-#include "llvm/Support/ErrorHandling.h"
```

Comment:
ErrorHandling.h is needed for llvm_unreachable, which you're otherwise just picking up transitively.

Reply:
Oops. I'll fix that in a follow-up commit.